authentication/wildfly-mods/arquillian.xml

```diff
@@ -0,0 +1,18 @@
+<arquillian xmlns="http://jboss.org/schema/arquillian"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="
+        http://jboss.org/schema/arquillian
+        http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+    <container qualifier="default" default="true">
+        <configuration>
+            <property name="jbossHome">${test.wildfly.home}</property>
+            <property name="javaVmArguments">${debugJvmArgs}</property>
+            <property name="serverName">wildfly</property>
+            <property name="hostName">${tck_hostname}</property>
+            <property name="username">${tck_username}</property>
+            <property name="password">${tck_password}</property>
+            <property name="httpPort">${tck_port}</property>
+            <property name="httpsPort">${tck_port_secure}</property>
```

Comment:
The only one of these that may work is `httpPort`, but it needs to change to `port` as the property name. The others won't do anything and the `username` and `password` could potentially break things as those are for the management client.